#goal definition
uint32 pose
uint32 posicao_inicial=1
uint32 posicao_pegar_container_superior=2
uint32 posicao_inicial_rotacionada=3
uint32 posicao_pegar_container_superior_rotacionado=4
---
#result definition
bool succeed
---
#feedback
float64 distance